### ISSUE TITLE
Fix release process

### ISSUE THREAD

**Issue by jamiehaywood** [Claudiohbsantos/github-material-icons-extension#43]
It looks like the latest release didn't make it to the Chrome store due to the release action failing. Might be worth failing the pipeline if a task fails?

**Comment by Claudiohbsantos** [comment]
Yeah, that's a little quirk of the current pipeline. Since I don't want to fail the pipeline if only one of the stores is updated the current solution ignores errors on the upload steps in the workflow. That's definitely not ideal but since nothing really depends on the results of these workflows I haven't invested too much time in splitting these tasks into properly isolated workflows that trigger each other.  Ultimately it always comes down to a manual check in the stores because the upload automations are kinda hacky.

I think the one solution would be to create 2 new workflows "update chrome store" and "update firefox store" that 
1. check latest release version in repo
2. check published version in store
3. attempts upload if behind

The current "Auto Update when upstream releases" workflow could then handle creating the release when necessary and triggering  the 2 new workflows. Either that or we could just run the update store workflows on a cron as well since they should be idempotent.

I'll probably not get to this for a bit because I think the main bottleneck will still be the fact that AFAIK store APIs are not really very friendly for this to be super robust so I still expect manual checks to be necessary every now and then. I'd be open for PRs if anyone is interested (though it is hard to test this kind of Action from a fork so I'd imagine this would be a tough contribution).

This particular release was stuck in the chrome store because some human intervention was needed to review the new permissions the last version added. It should be live now and following releases should be back to the fast-track automated reviews

**Comment by uncenter** [comment]
Seems like the recent release of v1.8.0 failed to get published on the Chrome Web Store as well, not sure if it is the same issue as here though.
